config/gatekeeper/apiextensions.k8s.io_v1_customresourcedefinition_assign.mutations.gatekeeper.sh.yaml

```diff
@@ -691,12 +1059,6 @@ spec:
             type: object
         type: object
     served: true
-    storage: true
+    storage: false
```

Comment:
what does this mean "storage: fales"

Reply:
As part of updating Gatekeeper, a `v1` version of this CRD was added and so the `v1` is being stored in Kubernetes' etcd instead of the `v1alpha1` now.